documentstore_migracao/utils/build_ps_package.py

```diff
@@ -0,0 +1,261 @@
+                    return False
+
+        return True
+
+    @property
+    def xml_fs(self):
+        return fs.open_fs(self.xml_folder)
+
+    @property
+    def img_fs(self):
+        return fs.open_fs(self.img_folder)
+
+    @property
+    def pdf_fs(self):
+        return fs.open_fs(self.pdf_folder)
+
+    @property
+    def out_fs(self):
+        return fs.open_fs(self.out_folder)
+
+    def copy(self, src_path, dst_path, src_fs=None, dst_fs=None):
+
+        if not src_fs:
+            src_fs = self.xml_fs
+
+        if not dst_fs:
+            dst_fs = self.out_fs
+        try:
+            copy.copy_file(src_fs, src_path, dst_fs, dst_path)
+
+            logging.info(
+                "Copy asset: %s to: %s" % (path.join(src_fs.root_path, src_path), path.join(dst_fs.root_path, dst_path))
+            )
+
+        except errors.ResourceNotFound as e:
+            logging.info(e)
+
+    def collect_xml(self, acron, xml):
+        issue_folder = path.basename(path.dirname(xml))
+
+        file_name_ext = path.basename(xml)
+
+        file_name, _ = path.splitext(file_name_ext)
+
+        target_folder = path.join(acron, issue_folder, file_name)
+
+        logging.info("Make dir package: %s" % target_folder)
+
+        self.out_fs.makedirs(target_folder, recreate=True)
+
+        xml_path = path.combine(acron, xml)
+
+        target_xml_path = path.join(acron, issue_folder, file_name, file_name_ext)
+
+        self.copy(xml_path, target_xml_path)
+        return issue_folder, file_name
+
+    def update_name(self, filename):
+
+        if filename.find("_") == 3:
+            name, ext = path.splitext(path.basename(filename))
+
+            return "%s%s%s%s" % (name[3:], "-", name[0:2], ext)
+        else:
+            return path.basename(filename)
+
+    def collect_pdf(self, acron, issue_folder, pack_name):
+
+        walker = Walker(filter=["*" + pack_name + "*.pdf"], max_depth=2)
+
+        pdf_path = path.join(self.pdf_fs.root_path, acron, issue_folder)
+
+        for pdf in walker.files(fs.open_fs(pdf_path)):
+
+            pdf_path = path.join(acron, issue_folder, path.basename(pdf))
+
+            target_pdf_path = path.join(
+                acron, issue_folder, pack_name, self.update_name(pdf)
+            )
+
+            self.copy(pdf_path, target_pdf_path, src_fs=self.pdf_fs)
+
+    def collect_img(self, acron, issue_folder, pack_name):
+
+        walker = Walker(filter=["*" + pack_name + "*"], max_depth=2, exclude_dirs=['html'])
+
+        img_path = path.join(self.img_fs.root_path, acron, issue_folder)
+
+        for img in walker.files(fs.open_fs(img_path)):
+
+            img_path = path.join(acron, issue_folder, path.basename(img))
+
+            target_img_path = path.join(
+                acron, issue_folder, pack_name, self.update_name(img)
+            )
+
+            self.copy(img_path, target_img_path, src_fs=self.img_fs)
+
+    def run(self):
```

Comment:
comentario sobre essa parte https://github.com/scieloorg/document-store-migracao/pull/116#issuecomment-494795995

Reply:
@cesarbruschetta a idéia é essa mesmo manter esse script de forma autônoma. 

"entendo que ela veio de quando vc esta fazendo um script a parte" sobre esse comentário não foi um erro e/ou esqueci de remove esse código. É intencional, quero que o script seja executado separadamente caso seja necessário.

A linguagem permiti isso com o trecho **if __name__ == "__main__":**, ref: https://www.geeksforgeeks.org/what-does-the-if-__name__-__main__-do/

Reply:
ok